DigitalOffice/Services/FileService/Validators/NewFileValidator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using FluentValidation;
+using FileService.Models;
+using System;
+
+namespace FileService.Validators
+{
+    public class NewFileValidator : AbstractValidator<FileRequest>
+    {
+        public NewFileValidator()
+        {
+            RuleFor(file => file.Name)
+                .NotEmpty()
+                .WithMessage("File must have a name")
+                .MaximumLength(244)
+                .WithMessage("File name is too long")
+                .Matches("^[A-Z 0-9][A-Z a-z 0-9]+$");
+            RuleFor(file => file.Content)
+                .Must(f => IsBase64Coded(f));
```

Comment:
Replace with method group.

Reply:
What happens if the Content is null?

Reply:
@MihailKulikov excuse me? Please, clarify what do you mean? What method group?
On the ContentIsNull -- good point. I should add null checks.

Reply:
You can write RuleFor(file => file.Content).Must(IsBase64Coded). It is little style fix, but it simplifies construction.

Reply:
Will check, thanks

Reply:
It's working and tests are passing. Thank you, Mikhail.